src/main/java/com/cgm/infolab/service/FromEntitiesToDtosService.java

```diff
@@ -0,0 +1,36 @@
+package com.cgm.infolab.service;
+
+import com.cgm.infolab.db.model.ChatMessageEntity;
+import com.cgm.infolab.db.model.RoomEntity;
+import com.cgm.infolab.db.model.UserEntity;
+import com.cgm.infolab.model.ChatMessageDto;
+import com.cgm.infolab.model.LastMessageDto;
+import com.cgm.infolab.model.RoomDto;
+import com.cgm.infolab.model.UserDto;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Fai caso al fatto che questa classe non ha alcuna dipendenza e non necessita di uno stato condiviso. I metodi sono delle semplici funzioni di traduzione. Significa che questa classe può tranquillamente essere una classe statica priva di stato che espone solo metodi statici. E non serve che vada in contesto di dependency injection tramite annotazione `@Service`. Così si semplifica anche nell'utilizzo.

Questi metodi si solito li chiamiamo `Mapper`